.env.development

```diff
@@ -2,5 +2,6 @@ VITE_APP_VERSION=0.0.1
 
 # ALA OIDC
 VITE_OIDC_AUTHORITY=https://auth-test.ala.org.au/cas/oidc
+VITE_TOKENS_API=https://api.test.ala.org.au/tokens
```

Comment:
I'm not sure if this .env.development file is used, doc.sh writes a .env.local file based on the values in config.ini, in AWS an .env.production file is written for all other environments

Reply:
This file has been removed